scripts/src/javatests/org/oppia/android/scripts/maven/RetrieveLicenseTextsTest.kt

```diff
@@ -539,6 +549,14 @@ class RetrieveLicenseTextsTest {
     return itemList
   }
 
+  private fun retrieveLongLicenseText(): String {
+    val charPool: List<Char> = ('a'..'z') + ('A'..'Z') + ('0'..'9') + ' '
```

Comment:
I am unable to understand this. Are we not considering any special characters? If no, then why?

Reply:
Changes this to include all chars from ASCII value 32 to 122.